trial/src/Trial.hs

```diff
@@ -9,7 +9,24 @@ Copyright: (c) 2020 Kowainik
 SPDX-License-Identifier: MPL-2.0
 Maintainer: Kowainik <xrom.xkov@gmail.com>
 
-Trial Data Type
+'Trial' data structure is 'Either'-like structure that stores tagged
```

Comment:
I guess this is only relevant to `TaggedTrial`, but the `Trial` don't have a tag...